packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -16291,6 +16291,99 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /purposeTemplates/{purposeTemplateId}/publish:
+    parameters:
+      - name: purposeTemplateId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      summary: Publish a Purpose Template
+      description: Publish a Purpose Template in Draft state
+      operationId: publishPurposeTemplate
```

Align with corresponding E-Service Template call
```suggestion
      summary: Publish a Purpose Template in Draft state
      description: Publish a Purpose Template that is in Draft state, transitioning it to Published state
```